sdk/eventhub/event-hubs/samples/expressSample/src/README.md

```diff
@@ -0,0 +1,64 @@
+---
+page_type: sample
+languages:
+  - typescript
+products:
+  - azure
+  - azure-event-hubs
+urlFragment: event-hubs-typescript-express-node
+---
```

Comment:
Sorry for the last minute request. Can you remove this metadata from the README? We're going to be adding more framework samples in the next week or so and may be moving this to another folder after it is merged.

Once this metadata is removed I'll merge!

Reply:
Removed.

Reply:
@maorleger has a PR out now for using a few of our packages with react:
https://github.com/Azure/azure-sdk-for-js/pull/12924

We'll be working on adding more this month :smile: